test/framework/src/main/java/org/elasticsearch/test/AbstractSerializingTestCase.java

```diff
@@ -72,4 +72,9 @@ protected boolean supportsUnknownFields() {
     protected ToXContent.Params getToXContentParams() {
         return ToXContent.EMPTY_PARAMS;
     }
+
+    protected boolean assertToXContentEquivalence() {
```

Comment:
Ah this was just missing in this base class. It does exist in the `AbstractXContentTestCase` base class.
Should we add this change back into master / 6.x separately after this PR is merged or just wait for when the ccr branches are merged? I have no strong opinion here.

Reply:
We discussed this offline. We will keep this in this branch for now, and it will come to 6.x/master when we integrate there.